src/Examine/LuceneEngine/Search/LuceneSearchQueryBase.cs

```diff
@@ -18,7 +18,9 @@ public abstract class LuceneSearchQueryBase : IQuery, INestedQuery
 
         internal readonly Stack<BooleanQuery> Queries = new Stack<BooleanQuery>();
         public BooleanQuery Query => Queries.Peek();
-        internal readonly List<SortField> SortFields = new List<SortField>();
+
+        private readonly List<SortField> _sortFields = new List<SortField>();
```

Comment:
Is there any reason to have a backing field instead of just `public List<SortField> SortFields {get;} = new List<SortField>();` ?

Reply:
Honestly, I don't know what my thinking was here...

I guess I was following the `QueryParser` property above for consistency which has the backing field, but in this case it doesn't make much sense...

I'll change it